tools/svg_recompress.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
+
+###############################################################################
+# Compress SVG files using svgo (https://github.com/svg/svgo)
+#
+# The following svgo plugins are disabled:
+# * convertPathData
+#     Though the resulting files are OK in any browsers, they're rendered in a
+#     weird way on Ubuntu files explorer.
+# * mergePaths
+#     Can result on wrong odd/even filling if some paths are overlapping.
+
+###############################################################################
+# Option variables
+
+opt_help=no
+opt_paths=""
+
+###############################################################################
+# Print the help message
+
+print_help() {
+    printf "Usage: `basename $0` [PATH...]
```

Comment:
nit: I would add `[OPTION...]` before `[PATH...]` for consistency but it's just me being pedantic :-)